src/App.vue

```diff
@@ -1,32 +1,81 @@
 <template>
-  <div id="app">
-    <div id="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
-    </div>
-    <router-view/>
-  </div>
-</template>
+  <v-app>
+    <v-app-bar app dark>
+      <v-toolbar-title>Localizar</v-toolbar-title>
 
-<style>
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-}
+      <v-spacer></v-spacer>
 
-#nav {
-  padding: 30px;
-}
+      <v-btn to="/" class="nav-btn">Home</v-btn>
```

Comment:
tutaj lepiej `to` zdefiniować jako obiekt z nazwą ścieżki

`<v-btn :to="{ name: 'rootPath' }" class="nav-btn">Home</v-btn> `

później jak bedziesz potrzebować zmienić ścieżkę to nazwe zostawisz a ścieżkę sobie zmienisz w routerze, a nie w każdym pliku

Reply:
Tu jest wykorzystywany name, tak? :D